### PR TITLE
Added Dice roll simulator

### LINKED ISSUE #439: Dice Rolling Simulator
**Describe the bug**
I can implement dice rolling simulator using tkinter in a efficient way.
You can replace [this](https://github.com/avinashkranjan/Amazing-Python-Scripts/tree/master/Dice-Roll-Simulator) or can add additional one using tkinter in your directory.

**Screenshots**
After executing it will look like this:
drs

### PR DESCRIPTION
**Pull request by mehabhalodiya**
# Description

We will need to import these modules:
- Tkinter: Imported to use Tkinter and make GUI applications.
- Image, Imagetk: Imported from PIL, i.e. Python Imaging Library. We use it to perform operations involving images in our UI.
- Random: Imported to generate random numbers.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by santushtisharma10 — APPROVED** [review]

**Comment by santushtisharma10** [comment]
@kaustubhgupta @antrikshmisri @tamanna-verma

File: Dice-Roll-Simulator/dice_roll_simulator.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+l0.pack()
+
+# adding label with different font and formatting
+l1 = tkinter.Label(root, text="Dice Rolling Simulator", fg = "white",
+        bg = "black",
+        font = "Helvetica 16 bold italic")
+l1.pack()
+
+# images
+dice = ['die1.png', 'die2.png', 'die3.png', 'die4.png', 'die5.png', 'die6.png']
```
Comment (kaustubhgupta) [inline Dice-Roll-Simulator/dice_roll_simulator.py:21]:
```suggestion
dice = ['./Dice-Roll-Simulator/imgs/die1.png', './Dice-Roll-Simulator/imgs/die2.png', './Dice-Roll-Simulator/imgs/die3.png', './Dice-Roll-Simulator/imgs/die4.png', './Dice-Roll-Simulator/imgs/die5.png', './Dice-Roll-Simulator/imgs/die6.png']
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@mehabhalodiya You again messed the image paths

File: Dice-Roll-Simulator/dice_roll_simulator.py (hunk 1 above)
Comment (antrikshmisri) [reply Dice-Roll-Simulator/dice_roll_simulator.py:21]: Or could even use list comprehension
```suggestion
dice_names = ['die1.png', 'die2.png', 'die3.png', 'die4.png', 'die5.png', 'die6.png']
dice_path = [os.path.join(os.getcwd() , 'imgs' , ele) for ele in dice_names]
```

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]
LGTM 💯

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**